hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/sigs/metadata/StateChildrenSigMetadataLookup.java

```diff
@@ -220,7 +226,19 @@ private SafeLookupResult<AccountSigningMetadata> lookupAccountByNumber(
         } else {
             final var key = account.getAccountKey();
             if (key == null || key.isEmpty()) {
```

Comment:
`key` should never be `null`. For the properties change-tracking, here we can do,
```
if (key.isEmpty()) {
    if (linkedRefs != null) {
        linkedRefs.link(fileNumbers.applicationProperties());
    }
    ....
  }
```

Reply:
Removed the unneeded null check for the `key`, will address the change tracking of the properties in a separate PR.